src/applications/mhv-secure-messaging/tests/e2e/pages/ContactListPage.js

```diff
@@ -79,6 +93,12 @@ class ContactListPage {
   clickBackToInbox = () => {
     cy.get(Locators.BACK_TO).click();
   };
+
+  verifyEmptyContactListAlert = () => {
+    cy.get(`#checkbox-error-message`)
+      .should(`be.visible`)
+      .and(`contain.text`, Alerts.CONTACT_LIST.EMPTY);
+  };
```

Comment:
I think we may want to add validating that correct checkbox receives proper focus after clicking `Save` and all checkboxes are blank.

Reply:
done